Pod/Classes/PINProgressiveImage.m

```diff
@@ -233,6 +233,7 @@ - (PINImage *)currentImageBlurred:(BOOL)blurred maxProgressiveRenderSize:(CGSize
                     currentImage = [PINImage imageWithCGImage:image];
                 }
                 CGImageRelease(image);
+                *imageProgress = progress;
```

Comment:
Do you need to check if imageProgress is non-nil? Or does the out keyword guarantee that?